package.json

```diff
@@ -102,6 +102,7 @@
     "@edsc/geo-utils": "^1.0.4",
     "@edsc/smart-handoffs": "^1.0.5",
     "@edsc/timeline": "^1.1.7",
+    "@mapbox/node-pre-gyp": "^1.0.11",
```

Comment:
This package is the successor the the previous one and is still being actively updated